tcms/bugs/tests/test_views.py

```diff
@@ -68,3 +80,60 @@ def test_reopen_a_closed_bug(self):
         self.assertRedirects(response, redirect_url)
         bug.refresh_from_db()
         self.assertTrue(bug.status)
+
+
+class TestNewBug(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        user_should_have_perm(cls.tester, 'bugs.add_bug')
+
+        cls.url = reverse('bugs-new')
+
+        cls.summary = 'A shiny new bug!'
+        cls.product = ProductFactory()
+        cls.version = VersionFactory(product=cls.product)
+        cls.build = BuildFactory(product=cls.product)
+        cls.post_data = {
+            'summary': cls.summary,
+            'reporter': cls.tester.pk,
+            'product': cls.product.pk,
+            'version': cls.version.pk,
+            'build': cls.version.pk,
+        }
+
+    def test_get_view(self):
+        response = self.client.get(self.url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.context['page_title'], 'New bug')
+        self.assertEqual(response.context['form_post_url'], reverse('bugs-new'))
+
+        form = response.context['form']
+
+        self.assertEqual(form.initial['reporter'].pk, self.tester.pk)
+        self.assertFalse(form.fields['version'].queryset.exists())
+        self.assertFalse(form.fields['build'].queryset.exists())
+        self.assertTemplateUsed(response, 'bugs/mutable.html')
+
+    def test_create_new_bug(self):
+        initial_bug_count = Bug.objects.count()
+        response = self.client.post(self.url, self.post_data)
+
+        bug_created = Bug.objects.last()
```

Comment:
assert that the newly created bug is what we want, e.g. summary  matches the test data that was sent.